bonjourmadame.el

```diff
@@ -55,7 +55,7 @@
 
 (defvar bonjourmadame--cache-dir (concat (or (getenv "XDG_CACHE_HOME") "~/.cache") "/bonjourmadame"))
 (defvar bonjourmadame--buffer-name "*Bonjour Madame*")
-(defvar bonjourmadame--base-url "http://ditesbonjouralamadame.tumblr.com")
+(defvar bonjourmadame--base-url "http://bonjourmadame.fr")
```

Comment:
Thanks for the PR !

Nevertheless, it should be `http://dites.bonjourmadame.fr` since `bonjourmadame.fr` redirects to `dites.bonjourmadame.fr`

Reply:
Ah I didn't check that.
I deleted my branch, do you want me to make a new PR or will you make the change?

Reply:
I'll take care of it :)

Thanks again!